PYME/IO/clusterIO.py

```diff
@@ -803,10 +803,10 @@ def get_file(filename, serverfilter=local_serverfilter, numRetries=3, use_file_c
         except (requests.Timeout, requests.ConnectionError) as e:
             # s.get sometimes raises ConnectionError instead of ReadTimeoutError
             # see https://github.com/requests/requests/issues/2392
-            logger.exception('Timeout on get file')
+            logger.exception('Timeout on get file %s' % url)
             logger.info('%d retries left' % (numRetries - nTries))
             if nTries == numRetries:
-                raise
+                raise e
```

Comment:
is this strictly needed? I thought that `raise` on it's own in an exception block just re-raised the exception that was being handled (i.e. didn't need the explicit exception argument), and was more idiomatic python.

Reply:
Not necessary, nor idiomatic. Same could be said for assigning the exception to an unused variable on 803. Would you like me to revert it?

Reply:
was more a  "wait, did I have this wrong" informational question. Suspect the `except as e` bit was left over from looking at some additional info in the exception object.

Reply:
Well, it's on me for changing something I didn't need to, sorry. Went ahead and took out the `as e`.